Library/Formula/h2o.rb

```diff
@@ -22,10 +22,9 @@ class H2o < Formula
 
   def install
     args = std_cmake_args
-    args << "."
     args << "-DWITH_BUNDLED_SSL=OFF"
 
-    system "cmake", *args
+    system "cmake", ".", *args
```

Comment:
Is the `.` definitely necessary? Think `std_cmake_args` may include it? Ignore me and :+1: if I'm wrong.